src/app/modules/verification/verification.model.ts

```diff
@@ -128,7 +150,27 @@ const compileVerificationModel = (db: Mongoose): IVerificationModel => {
     ).exec()
   }
 
-  VerificationSchema.statics.resetField = async function (
+  VerificationSchema.statics.incrementPaymentFieldRetries = async function (
+    transactionId: string,
+  ): Promise<IVerificationSchema | null> {
+    return this.findOneAndUpdate(
+      {
+        _id: transactionId,
+      },
+      {
+        $inc: {
+          'paymentField.hashRetries': 1,
+        },
+      },
+      {
+        new: true,
+        runValidators: true,
+        setDefaultsOnInsert: true,
+      },
+    ).exec()
```

Comment:
supernit: Not sure if this is a good idea, but it may be possible to abstract the arguments for all of these functions out, to avoid repeating the combinations (after all, it's just the prefix of `paymentField` or `fields.$` that changes). Nonetheless, not critical.

Reply:
Done in [8b1eed0](https://github.com/opengovsg/FormSG/pull/6003/commits/8b1eed09b66bddbb99cdc30b40f25646680e70ab).